File: src-tauri/src/models/seed.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Serialize, Deserialize)]
#[serde(rename_all = "camelCase")]
pub struct SeedResult {
    pub teachers_created: usize,
    pub subjects_created: usize,
    pub classes_created: usize,
    pub rooms_created: usize,
    pub assignments_created: usize,
}

File: src/types/seed.ts
export interface SeedResult {
  teachersCreated: number;
  subjectsCreated: number;
  classesCreated: number;
  roomsCreated: number;
  assignmentsCreated: number;
}

export type SchoolType = "gymnasium" | "grundschule" | "mittelschule" | "vollstaendig";
